src/components/Chat/Chat.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import Message from '../Message/Message.js';
+import './Chat.css';
+class Chat extends Component {
+  state = {
+    messageInput: '',
+    messages: []
+  };
+  handleOnChange = event => {
+    this.setState({
+      messageInput: event.target.value
+    });
+    if (event.key === 'Enter') {
+      this.sendMessageOnEnter();
+    }
+    ///[...this.state.messages, ...[1,2,3]
+  };
+  // changeInputMessage = () => {};
+  sendMessageOnEnter = () => {
+    const newMess = this.state.messageInput;
+    // this.changeInputMessage();
+    this.setState(state => ({
+      messages: [...this.state.messages, ...[newMess]],
```

Comment:
`messages: [...this.state.messages, ...[{text: newMess]}], `
` messageInput: '' `